src/Data/Generics/Product/Positions.hs

```diff
@@ -147,6 +165,37 @@ coerced = coerce
 instance {-# OVERLAPPING #-} HasPosition f (Void1 a) (Void1 b) a b where
   position = undefined
 
+instance
+  ( Generic s
+  , Generic t
+  , ErrorUnless i s (0 <? i && i <=? Size (Rep s))
+  , GLens (HasTotalPositionPSym i) (CRep s) (CRep t) a b
+  , HasTotalPositionP i (CRep s) ~~ 'Just a
+  , HasTotalPositionP i (CRep t) ~~ 'Just b
+  , HasTotalPositionP i (CRep (Indexed s)) ~~ 'Just a'
+  , HasTotalPositionP i (CRep (Indexed t)) ~~ 'Just b'
```

Comment:
Should these `Indexed` ones be removed from here? I think they're redundant (same for `HasPosition_`), given that `Infer` is no longer using them.

Reply:
Are you proposing to remove all `HasTotal*** ... (CRep (Indexed _)) ...`?